test/glue/pulp/consumer_test.rb

```diff
@@ -184,6 +184,10 @@ def test_uninstall_package_group
 
     def test_install_consumer_errata
       erratum_id = RepositorySupport.repo.errata_json.select { |errata| errata['id'] == 'RHEA-2010:0002' }.first['id']
+      profile = [{"vendor" => "FedoraHosted", "name" => "elephant",
+                  "version" => "0.3", "release" => "0.8",
+                  "arch" => "noarch", :epoch => ""}]
```

Comment:
Pulp started returning a 400 bad response if there was no package profile present which isn't really an error (but manifests so on our side). So I am not sure that anything needs to be filed?